src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -256,54 +256,58 @@ private async Task<CreationResultStatus> EnterInstallFlow()
 
             // In future we might want give user ability to pick IManagerSourceProvider by Name or GUID
             var managedSourceProvider = EnvironmentSettings.SettingsLoader.TemplatesSourcesManager.GetManagedProvider(GlobalSettingsTemplatesSourcesProviderFactory.FactoryId);
-            foreach (var identifier in _commandInput.ToInstallList)
+            foreach (var unexpandedInstallRequest in _commandInput.ToInstallList)
             {
-                var splitByColons = identifier.Split(new[] { "::" }, StringSplitOptions.RemoveEmptyEntries);
-                SemanticVersion version = null;
-                if (splitByColons.Length > 1 && SemanticVersion.TryParse(splitByColons[1], out var parsedVersion))
+                foreach (var expandedInstallRequest in InstallRequestPathResolution.Expand(unexpandedInstallRequest, EnvironmentSettings))
```

Comment:
I'm not sure if we want expand behavior in CLI. Should we check it with Kathleen?